### ISSUE TITLE
Issue when importing tag libraries

### ISSUE THREAD

**Issue by donbellamy** [jcberquist/commandbox-cfformat#89]
When using this code:

```
<cfimport prefix="pbn" taglib="/pbn/tags/" />

<div class="back mb-1">
    <pbn:link-returnto href="xxx" text="xxx" />
</div>
```

I am receiving the following error:

&lt;pbn&gt; on line 4 does not match closing &lt;/div&gt; on line 5.

**Comment by jcberquist** [comment]
This looks like an issue when running `cfformat tag-check`, correct? In any case, I have added some fixes for prefixed custom tags in `v0.15.9` that should fix this. Thanks for reporting it.

**Comment by donbellamy** [comment]
Yes, when using tag-check.  Thanks!
